UefiTestingPkg/AuditTests/BootAuditTest/UEFI/BootAuditTestApp.c

```diff
@@ -147,18 +260,19 @@ OsIndicationsSupportTest (
     SysPrep testing
 
 */
-static
+STATIC
 UNIT_TEST_STATUS
 EFIAPI
 VariableLockedTestTest (
   IN UNIT_TEST_CONTEXT  Context
   )
 {
-  BASIC_TEST_CONTEXT  *Btc;
-  EFI_STATUS          Status;
-  UINTN               DataSize;
-  UINT32              Attributes;
-  UINT64              Data = 0x1122334455667788;
+  BASIC_TEST_CONTEXT            *Btc;
+  EFI_STATUS                    Status;
+  UINTN                         DataSize;
+  UINT32                        Attributes;
+  UINT64                        Data = 0x1122334455667788;

```

Comment:
Would we still run into the issue with invalid parameters if we try to set variable (i.e. sysprep) with this value?

Reply:
SysPrep####, PlatformRecovery####, and Driver#### have a different execution path in this test app (via checking the test context OptionType) and don't use this Data variable.

Reply:
Thanks for the explanation.